security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -0,0 +1,44 @@
+//OpenShift File Integrity Operator Release Notes
+[id="file-integrity-operator-release-notes"]
+= File Integrity Operator release notes
+:context: file-integrity-operator-release-notes-v0
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+The File Integrity Operator lets {product-title} administrators describe the required compliance state of a cluster and provides them with an overview of gaps and ways to remediate them.
+
+These release notes track the development of the File Integrity Operator in the {product-title}.
+
+For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
+
+[id="file-integrity-operator-release-notes-0-1-21"]
+== OpenShift File Integrity Operator 0.1.21
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.21:
+
+* link:https://access.redhat.com/errata/RHBA-2021:4631[RHBA-2021:4631 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
+
+[id="file-integrity-operator-0-1-21-new-features-and-enhancements"]
+=== New features and enhancements
+
+* There are now informational metrics related to the FileIntegrity scan results and processing. The metrics exposed to the monitoring dashboard on the console are labeled with the prefix of `file_integrity_operator_`.
++
+* There is a default PrometheusRule provided in the operator namespace that alerts with a warning if a node has an integrity failure for more than 1 second.
++
+* Some dynamic MCO and CVO related filepaths have been excluded from the default AIDE policy to help prevent false positives during node updates. The excluded paths are:
+ - /etc/machine-config-daemon/currentconfig
+ - /etc/pki/ca-trust/extracted/java/cacerts
+ - /etc/cvo/updatepayloads
+ - /root/.kube
++
+* The AIDE daemon process has stability improvements and is more resilient to errors during database re-init.
```

Comment:
Are we comparing deltas between an FIO upstream version to this released version (0.1.22)?  The terms "stability improvements" and "more resilient" hint at a delta between versions.

Reply:
I can update it to say that there are stability improvements since v0.1.16, which was the release prior to the current.